api.swagger.yaml

```diff
@@ -9776,6 +9776,10 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/title-uuid-list-item-common-schema'
+            read_only:
+              description: 'Users who are flagged as read-only are not able to create, update, or delete content.'
+              type: boolean
+              example: false
```

Comment:
Can we make this property read-only in case the schemas move around the future

Reply:
done